docs/_includes/nav.html

```diff
@@ -1,10 +1,7 @@
-<nav>
-  <ul>
-    {% for item in site.data.navigation %}
-    <li>
-      <a href="{{item.link | relative_url }}" {% if page.url==item.link %}style="color: red;" {% endif
-        %}>{{item.name}}</a>
-    </li>
-    {% endfor %}
-  </ul>
-</nav>
\ No newline at end of file
+{% for item in site.data.navigation %}
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" aria-current="page" href="{{ item.link | relative_url }}" {% if page.url==item.link %}style="color: #9A3324;" {% endif%}><u>{{item.name}}</u></a>
```

Comment:
Could use a bootstrap link color class here in the conditional instead of a hex value. https://getbootstrap.com/docs/5.1/helpers/colored-links/

Reply:
That doesn't work. Probably there is a class attribute and it won't apply another class attribute, I have to leave as style. I removed the <u> tag and opted for CSS styling for underline.